### PR TITLE
Adding support for Maya 2025

### PR DESCRIPTION
**Pull request by iamsleepy**
Adding support for Maya 2025.

No building steps from 7 to 9(except for building a wheel).

### CODE REVIEW

File: pymel/versions.py [hunk 1]
```diff
@@ -184,6 +184,9 @@ def bitness():
 v20023_2 = 20230200
 v20023_3 = 20230300
 v20024 = 20240000
+v20024_1 = 20240100
+v20024_2 = 20240200
+v2025 = 20250000
```
Comment (chadrik) [inline pymel/versions.py:189]: unbelievably, these 2023 and 2024 versions seem to have an extra 0.  can you correct this, like so:

```python
v20023_1 = 20230100  # bad name.  keeping for backwards compatibility
v20023_2 = 20230200  # bad name.  keeping for backwards compatibility
v20023_3 = 20230300  # bad name.  keeping for backwards compatibility
v2023_1 = 20230100
v2023_2 = 20230200
v2023_3 = 20230300
v20024 = 20240000
v20024 = 20240000  # bad name.  keeping for backwards compatibility
v2024 = 20240000
v2024_1 = 20240100
v2024_2 = 20240200
v2025 = 20250000
```

File: pymel/core/datatypes.py [hunk 2]
```diff
@@ -3337,7 +3337,7 @@ class Time(Unit):
     def _inCast(cls, x):
         return cls(x)._data
 # ------ Do not edit below this line --------
-    if versions.current() >= 20240000:
+    if versions.current() >= versions.v20024:
```
Comment (chadrik) [inline pymel/core/datatypes.py:3340]: let's use the corrected constant for this: `v2024`

**Review by chadrik — COMMENTED** [review]
@iamsleepy  If you can set your repo to allow me to push to it then we can collaborate on this PR.

**Comment by iamsleepy** [comment]
@chadrik I've added you as a collaborator.

**Comment by chadrik** [comment]
I updated the changelog and addressed a few small issues.   I think this is ready to merge, but I had a quick question about `maya.cmds.matrix`.  This generated pymel code indicates that this function may have been removed.  Do you have a chance to double check this?  Sorry, I don't have a copy of Maya 2025 to test with.

**Comment by iamsleepy** [comment]
It seems have been removed in Maya 2025.
`import maya.cmds as cmds

cmds.matrix
# Error: AttributeError: file <maya console> line 3: module 'maya.cmds' has no attribute 'matrix'
`
Compare to Maya 2024
`import maya.cmds as cmds

cmds.matrix
`

**Comment by Shapemob** [comment]
Any news ?

**Comment by JasonGilholme** [comment]
@chadrik Any more thoughts on this?

**Comment by StefanHabel** [comment]
Thank you for all your work on this!

Minor note: `CHANGELOG.rst` appears to contain two sections for `Version 1.5.0`, which could be merged into one.

**Pull request merged**